api/authn/v1/authn.proto

```diff
@@ -52,3 +60,20 @@ message CallbackResponse {
   // context the user will be redirected.
   string access_token = 1;
 }
+
+message CreateTokenRequest {
+  option (clutch.api.v1.redacted) = true;
+
+  // The subject to issue this token for.
+  string subject = 1;
+
+  // The duration until this token should expire.
+  google.protobuf.Duration expiry = 2;
```

Comment:
what is default? i think we'll want no expiration as default or at least a possibility.